vault.yaml

```diff
@@ -53,12 +54,31 @@ pipeline:
       install -m644 -D "./vault.confd" "${{targets.destdir}}/etc/conf.d/vault"
 
       install -m755 -D ./vault/bin/vault "${{targets.destdir}}/usr/sbin/vault"
+      setcap cap_ipc_lock=+ep  "${{targets.destdir}}/usr/sbin/vault"
+      # Binary doesn't require root, so let's move it out of sbin - for a temporary period have it
+      # in both locations. We can later delete the sbin one.
+      mkdir "${{targets.destdir}}/usr/bin"
+      chmod 755 "${{targets.destdir}}/usr/bin"
+      ln -s /usr/sbin/vault "${{targets.destdir}}/usr/bin/vault"
```

Comment:
Is it in /usr/sbin/vault for compatibility with an upstream Helm chart? If so we might want to make that symlinking part of a separate compat package.

Reply:
The upstream Helm chart doesn't work at all with our current image. This PR adds the entrypoint package, which effectively adds Helm compatibility.

This change is separate though - I just wanted to move the binary out of sbin, as I don't think it should be there.

Reply:
I think we should just move it to `/usr/bin/vault`, if it sounds like we don't need to worry about breaking anybody since this was already not functional. WDYT?

Reply:
It was functional, just not compatible with the helm chart or upstream image. There was actually another breakage where the config file wasn't being picked up.

Reply:
I guess I'm being stupid though, that's literally a one char change to the apko that we can put through immediately.

Reply:
Ok, created https://github.com/chainguard-images/images/pull/731 and will resolve this conversation.